.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,397 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+name: ci
+
+on:
+  # Note: Jobs are skipped when ONLY Markdown (*.md) files are changed
+  push:
+    branches:
+      - master
+      - RB-*
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    branches:
+      - master
+      - RB-*
```

Comment:
To correctly understand all the pull requests must use the CI build. It should not be limited to pull requests with a specific branch name.

Reply:
Good point! The goal here was to prevent CI running for updates to older branches, bit that did limit the use quite a bit. I've changed it to explicitly exclude v0 and v1, but allow all others, which will behave with forks much more naturally now.